src/components/PaymentStripe/cardSection.js

```diff
@@ -0,0 +1,117 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { CardElement, useElements, useStripe } from '@stripe/react-stripe-js';
+import { useTheme, useUniqueIdentifier } from '../../libs/hooks';
+import RenderIf from '../RenderIf';
+import Label from '../Input/label';
+import HelpText from '../Input/styled/helpText';
+import ErrorText from '../Input/styled/errorText';
+import StyledContainer from './styled/container';
+
+const CardSection = props => {
+    const {
+        label,
+        hideLabel,
+        bottomHelpText,
+        error,
+        className,
+        style,
+        iconStyle,
+        disabled,
+        onChange,
+        onFocus,
+        onBlur,
+    } = props;
+    const stripe = useStripe();
+    const elements = useElements();
+    const cardElementId = useUniqueIdentifier('card-element');
+    const theme = useTheme().rainbow;
+    const cardElementOptions = useMemo(
+        () => ({
+            style: {
+                base: {
+                    iconColor: theme.palette.text.main,
+                    fontFamily: '"Lato", Arial, sans-serif',
+                    backgroundColor: theme.palette.background.main,
+                    color: theme.palette.text.main,
+                    fontSize: '1.1rem',
+                    '::placeholder': {
+                        color: theme.palette.text.header,
+                        fontVariant: 300,
+                    },
+                },
+            },
+            iconStyle,
+            disabled,
+        }),
+        [theme, iconStyle, disabled],
+    );
+
+    const handleChange = event => {
+        if (stripe && elements) {
+            const payment = {
+                stripe,
+                element: elements.getElement(CardElement),
+                empty: event.empty,
+                complete: event.complete,
+                brand: event.brand,
```

Comment:
maybe a good opportunity to improve the names of stripe props
- empty -> isEmpty 
- complete -> isComplete
- brand -> cardBrand